app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SessionsController < ApplicationController
+
+  def new
+
+  end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
```

Comment:
Ich habe eine Frage zu dieser `user" Variable: In den anderen Controllers (`user` und `article`) ist dies eine Instanzvariable mit "@" (`@user` z.B.). Warum ist diese Variable "normal"? Ist es, weil ich sie nirgendwo anders in meinem Code nutze (außerhalb dieser Funktion)?

Reply:
ja genau